packages/playwright-ct-svelte/registerSource.mjs

```diff
@@ -54,14 +55,22 @@ window.playwrightMount = async (component, rootElement, hooksConfig) => {
   for (const hook of /** @type {any} */(window).__pw_hooks_before_mount || [])
     await hook({ hooksConfig });
 
-  const wrapper = new componentCtor({
+  const svelteComponent = /** @type {SvelteComponent} */ (new componentCtor({
     target: rootElement,
     props: component.options?.props,
-  });
+  }));
+  rootElement[svelteComponentKey] = svelteComponent;
 
   for (const hook of /** @type {any} */(window).__pw_hooks_after_mount || [])
-    await hook({ hooksConfig });
+    await hook({ hooksConfig, svelteComponent });
 
   for (const [key, listener] of Object.entries(component.options?.on || {}))
-    wrapper.$on(key, event => listener(event.detail));
+    svelteComponent.$on(key, event => listener(event.detail));
+};
+
+window.playwrightUnmount = async (element, rootElement) => {
+  const svelteComponent = /** @type {SvelteComponent} */ (rootElement[svelteComponentKey]);
+  svelteComponent.$destroy();
```

Comment:
Should we throw a meaningful error when I call `unmount(someRandomLocator)` instead of passing the component that I mounted before?